home/default.nix

```diff
@@ -121,6 +121,7 @@
     # https://nixos.asia/en/direnv
     direnv = {
       enable = true;
+      package = config.nix.package;
```

Comment:
Isn't it nix-direnv.package instead of direnv.package

Reply:
Ah, yes, this PR broke stuff. 

But your change is also not correct?

<img width="288" alt="image" src="https://github.com/user-attachments/assets/de855088-31f8-416b-8b65-c137d5f92ecf">

<img width="550" alt="image" src="https://github.com/user-attachments/assets/56088b94-1a80-4ce2-bbb9-78f2faa08b8e">

Reply:
It is already using `config.nix.package`. I'll revert this PR for now.

Reply:
Actually, this was my fault for not paying attention to Rana's actual change:

![image](https://github.com/user-attachments/assets/47a5418b-77f1-43d4-8293-74ff166326b1)

Reply:
@srid the nix-direnv configuration above is from nixos modules and not from home-manager. 

This has to be fixed in home-manager to match nixos module.